Quotas.qmd

```diff
@@ -0,0 +1,643 @@
+  <br>
+  ³ This includes, among others, a newly generated token and logging in to the user panel. Please refer to: <a href= "https://identity.dataspace.copernicus.eu/auth/realms/CDSE/account/#/security/device-activity" target="_blank">Device Activity</a> and see the number of signed in devices  - It is not possible to increase this number to a bigger value than 100, with a paid plan. Each session starts when the user generates a new token and use it for the downloading process. One session ends when the token reaches its expiry time (a token stays active for 10 min).
+  <br>
+  ⁴ After reaching this limit, the Access Token must either be refreshed by using the <a href= "https://documentation.dataspace.copernicus.eu/APIs/OData.html#product-download" target="_blank">Refresh Token</a>. 
+  <br>
+  ⁵ Anytime within 60 minutes after the access token is generated.
+  <br>
+  ⁶ As long as any other limit(s) are not breached.
+  <br>
+  ⁷ Extensions are available via CREODIAS offering (see 3.2.3 Product Generation Service of the <a href= "https://documentation.dataspace.copernicus.eu/_docs/CDSE-SDE-TSY_Service%20Description%20and%20Evolution_1.1.pdf" target="_blank">Roadmap</a>).
+  <br>
+  ⁸ Similar principles apply for all SentinelHub APIs while the differences and details are covered in the <a href= "https://documentation.dataspace.copernicus.eu/APIs/SentinelHub/Overview/ProcessingUnit.html" target="_blank">Processing Unit</a> section of our documentation. Note that there are APIs that are not available to Copernicus General Users such as Sentinel Hub Batch Processing API.
+  <br>
+  ⁹ Technical limitation may be applied to maintain platform stability.
+  <br>
+  ¹⁰ After reaching this monthly transfer limit, the maximum bandwidth drops to 1MB/s and the number of concurrent connections drops to 1.
+  <br>
+  ¹¹ Only enabled for openEO synchronous execution requests (`POST /result`) and batch execution requests (`POST /{job_id}/result`).
+  <br>
+  ¹² The maximum number of simultaneous
+production orders running in parallel, i.e. the orders in “In progress” status.
+  <br>
+  ¹³ 1 request per 5 seconds.
+  <br>
+  <br>
+  These limitations apply to the Copernicus General user type. Each Copernicus privileged user type (Copernicus Service User, Copernicus International Partner...) may be given other quotas. Please contact Copernicus Data Space Ecosystem Support Team via <a href= "https://helpcenter.dataspace.copernicus.eu/hc/en-gb/requests/new" target="_blank">Submit a request</a> for more information.
+  <br>
+  <br>
+  The following limitations and quotas are monthy and reset on the first day of every month:<br><br>
+  Requests per month<br>
+  Requests per minute<br>
+  Processing units (PU) per month<br>
+  Processing units (PU) per minute<br>
+  Monthly transfer limit (TB) (IAD¹)¹⁰<br>
+  Number of concurrent orders limit (DAD²)<br>
+  Monthly transfer limit (TB) (DAD²)<br>
+  Processed Products per month<br>
+  Credits per month<br><br>
```

Comment:
Thank you for removing the numbering but I also suggested using bullets instead. What do you think of that?

Reply:
Sure. Oh I misunderstood that. Sound good, I'm implementing bullets in my upcoming commit.